torchani/neurochem/__init__.py

```diff
@@ -262,9 +262,13 @@ def load_model_ensemble(species, prefix, count):
     return Ensemble(models)
 
 
+# Future: Delete BuiltinsAbstract in a future release, it is DEPRECATED
 class BuiltinsAbstract(object):
     """Base class for loading ANI neural network from configuration files.
 
+    This class is part of an old API. It is DEPRECATED and may be deleted in a
```

Comment:
Could you put this into a `.. warning::` block? See https://github.com/aiqm/torchani/blob/master/examples/nnp_training.py#L197 for example

Same for other `Builtins*` classes.

Reply:
Fixed in d758321